sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/checks/security/android/AndroidExportedContentPermissionsCheck.java

```diff
@@ -34,11 +34,11 @@
   private final XPathExpression xPathExpression = XPathBuilder
     .forExpression("/manifest/application/*" +
       "[" +
-      " (self::provider or self::activity or self::service)" +
+      " (self::provider or self::activity or self::activity-alias or self::service)" +
       "and" +
       " not(@n:exported='false')" +
       "and" +
-      " (@n:exported='true' or intent-filter)" +
+      " ((@n:exported='true' and not(intent-filter)) or intent-filter/action)" +
```

Comment:
self::activity-alias without permissions could be fine depending on the parent activity settings